cmd/experimental/kueue-viz/frontend/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
```

Comment:
What will the e2e tests look like?  Will it be clicking through ui?

Reply:
No. We will run cypress in prow.

Reply:
I am maintaining Cypress test in another repository (hosted in GitHub actions).
The result looking is https://github.com/kubeflow/katib/actions/runs/13820223041/job/38663514145#step:6:1.

In the Prow's case, I think we can see a similar execution and results as well.